packages/components/CONTRIBUTING.md

```diff
@@ -156,7 +156,7 @@ function Example(
 
 A couple of good examples of how hooks are used for composition are:
 
-- the `Card` component, which builds on top of the `Surface` component by [calling the `useSurface` hook inside its own hook](/packages/components/src/card/card/hook.js);
+- the `Card` component, which builds on top of the `Surface` component by [calling the `useSurface` hook inside its own hook](/packages/components/src/card/card/hook.ts);
```

Comment:
It looks like there are more references left to `Card` files ending in `*.js` in this file, that also need updating